packages/nx-plugin/src/executors/cli/executor.unit.test.ts

```diff
@@ -34,26 +22,40 @@ describe('runAutorunExecutor', () => {
 
   beforeEach(() => {
     vi.unstubAllEnvs();
+    executeProcessSpy.mockResolvedValue({
+      code: 0,
+      stdout: '',
+      stderr: '',
+      date: new Date().toISOString(),
+      duration: 100,
+    });
   });
 
   afterEach(() => {
     loggerWarnSpy.mockReset();
     loggerInfoSpy.mockReset();
+    executeProcessSpy.mockReset();
   });
 
   afterAll(() => {
     Object.entries(processEnvCP).forEach(([k, v]) => (process.env[k] = v));
   });
   /* eslint-enable functional/immutable-data, @typescript-eslint/no-dynamic-delete */
 
-  it('should call execSync with return result', async () => {
+  it('should call executeProcess with return result', async () => {
     const output = await runAutorunExecutor({}, executorContext('utils'));
     expect(output.success).toBe(true);
     expect(output.command).toMatch('npx @code-pushup/cli');
-    // eslint-disable-next-line n/no-sync
-    expect(execSync).toHaveBeenCalledWith(
-      expect.stringContaining('npx @code-pushup/cli'),
-      { cwd: MEMFS_VOLUME },
+    expect(executeProcessSpy).toHaveBeenCalledWith(
+      expect.objectContaining({
+        command: 'npx',
+        args: expect.arrayContaining(['@code-pushup/cli']),
+        cwd: MEMFS_VOLUME,
+        observer: expect.objectContaining({
+          onError: expect.any(Function),
+          onStdout: expect.any(Function),
+        }),
+      }),
     );
```

Comment:
See previous comment.

```suggestion
    expect(executeProcessSpy).toHaveBeenCalledWith({
      command: 'npx',
      args: expect.arrayContaining(['@code-pushup/cli']),
      cwd: MEMFS_VOLUME,
      observer: {
        onError: expect.any(Function),
        onStdout: expect.any(Function),
      },
    });
```